proxies/live/apiproxy/policies/AccessEntity.AppDetails.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Since we don't use this policy should we not remove it to avoid future confusion?
Same with the policies AddClientCredentials and ExtractClientId